### DISCUSSION TITLE
Regarding ecr Package

### DISCUSSION THREAD

**Discussion by SudoWodo** [SudoWodo/StochOptim#12] [Q&A] [answered]
I am not able to understand how to obtain the number of function call made by the optimizer. It is required so that It can be printed in the output by global wrapper. Can anyone help?

Thank you

**Comment by nashjc** [comment, marked as answer]
Most solvers return counts of "iterations", "functions" etc. BUT ... they often use different names. 

Another approach is to create an environment that stores data and assign the counts there, but this does add overhead.

JN
